client/app/ip/ip/firewall/game/ip-ip-firewall-game.constants.js

```diff
@@ -0,0 +1,19 @@
+export const IP_MITIGIATION_RULE_PROTOCOL_GAMES = {
+  arkSurvivalEvolved: 'Ark Survival Evolved',
+  arma: 'Arma',
+  gtaMultiTheftAutoSanAndreas: 'GTA Multi Theft Auto San Andreas',
+  gtaSanAndreasMultiplayerMod: 'GTA San Andreas Multiplayer Mode',
+  hl2Source: 'Half life/source',
+  minecraftPocketEdition: 'Minecraft Pocket Edition',
+  minecraftQuery: 'Minecraft Query',
+  mumble: 'Mumble',
+  other: 'Others',
+  rust: 'Rust',
+  teamspeak2: 'Teamspeak 2',
+  teamspeak3: 'Teamspeak 3',
+  trackmaniaShootmania: 'Trackmania Shootmania',
```

Comment:
It seems like most of it could be generated with a `_.startCase`. Maybe we could use this as default behaviour and add special cases ? That way we possibly don't have anything to do if a new protocol is added. What do you think ?

Reply:
It works for all the values except `Hl2Source => Half life/source` and `mod => mode`. 
Game names might not be exact values here.
We can use `_.startCase` in places where same names are used but different versions i think.

Reply:
I had in mind something like 
```suggestion
  return IP_MITIGIATION_RULE_PROTOCOL_GAMES[protocol] || _.snakeCase(protocol
```
with  IP_MITIGIATION_RULE_PROTOCOL_GAMES containing the mapping for special cases

Reply:
Okay i have done this change